TWLight/applications/templates/applications/application_evaluation.html

```diff
@@ -221,21 +221,41 @@ <h2>{% trans 'Application' %}</h2>
   {% comment %}Translators: This is the title of the section of an application page which contains the Recent Applications made by the applicant{% endcomment %}
   <h2>{% trans 'Recent Applications' %}</h2>
   
-  <div class="row">
-    {% for app in recent_apps %}
-      <div class="col-xs-12 col-sm-4">
-        {% if app.partner.coordinator == request.user or app.partner.coordinator == app.editor.user %}
-          <a href="{{ app.get_absolute_url }}">
-            <strong>{{ app.partner }}</strong>
-          </a>
-        {% else %}
-          <strong>{{ app.partner }}</strong>
-        {% endif %}
-      </div>
-      <div class="col-xs-12 col-sm-8">
-        <span class="label label{{ app.get_bootstrap_class }} label-lg">{{ app.get_status_display }}</span>
-      </div>
-    {% endfor %}
+  <div class="table-responsive">
+    <table class="table table-hover table-bordered">
```

Comment:
I feel that a table without borders looks better. Could you please remove the `table-bordered` class?

Reply:
removed borders from table 👍